doc/api/assert.md

```diff
@@ -147,6 +147,86 @@ try {
 }
 ```
 
+## Class : assert.CallTracker
+
+### `new assert.CallTracker()`
+<!-- YAML
+added: REPLACEME
+-->
+
+Returns a [`CallTracker`][] object.
+
+```js
+const assert = require('assert');
+
+const tracker = new assert.CallTracker();
+
+function func() {}
+
+callsfunc = tracker.calls(func);
+// Callsfunc() must be called exactly 1 time before tracker.verify()
+
+callsfunc();
+
+process.on('exit', () => {
+  tracker.verify();
+  // Verifies if all tracker.calls() functions have been called exact times
+});
+```
+
+### `assert.CallTracker.calls([fn][, exact])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `fn` {Function} **Default** A no-op function.
+* `exact` {number} **Default** `1`
+
+Returns a function that wraps `fn`. The wrapper function is expected to be
+called exactly `exact` times before the process exits. If the function is
+not called exactly `exact` times before [`assert.CallTracker.verify()`][] is
+called, [`assert.CallTracker.verify()`][] throws an error.
```

Comment:
The current sentence is fine, but the use of `assert.CallTracker.verify()` feels a bit redundant.

```suggestion
called, an exception is thrown.
```